pkgs/data/icons/tela-icon-theme/default.nix

```
@@ -0,0 +1,34 @@
+{ fetchFromGitHub, gtk3, hicolor-icon-theme, lib, stdenv }:
+
+stdenv.mkDerivation rec {
+  pname = "tela-icon-theme";
+  version = "2021-01-21";
+
+  src = fetchFromGitHub {
+    owner = "vinceliuice";
+    repo = pname;
+    rev = version;
+    sha256 = "0gphy4aq2qjcg79k6rc0q5901mn3q76qhckn5vxvmypn9n3lb9ph";
+  };
+
+  nativeBuildInputs = [ gtk3 ];
+
+  propogatedBuildInputs = [ hicolor-icon-theme ];
+
+  dontDropIconThemeCache = true;
+
+  installPhase = ''
+    runHook preInstall
+    patchShebangs install.sh
+    mkdir -p $out/share/icons
+    ./install.sh -a -d $out/share/icons
```

```suggestion
    ./install.sh -a -d $out/share/icons
    jdupes -l -r $out/share/icons
```
Use `jdupes` to make relative symlinks for duplicates w/o prompting, and for every directory, process its subdirectories too.